### PR TITLE
24474: Fixed incorrect handling of name property in DAR address handl…

### PR DESCRIPTION
**Pull request by cmoesgaard**
…er when lookup failed, causing detail creation to raise exception

- [ ] Have you updated the documentation?
- [x] Have you performed a smoke test of the application?
- [x] Have you written tests for your changes?
- [x] Have you updated the release notes?

### CODE REVIEW

File: backend/tests/test_address_handlers.py [hunk 1]
```diff
@@ -101,6 +101,36 @@ def test_get_lora_properties(self, mock):
...
+            'value': '300f16fd-fb60-4fec-8a2a-8d391e86bf3f'
+        }
+
+        # Act
+        actual_name = address_handler.name
+        actual_href = address_handler.href
+        actual_mo_address = address_handler.get_mo_address_and_properties()
+
+        # Assert
+        self.assertEqual(expected_name, actual_name)
```
Comment (danchr) [inline backend/tests/test_address_handlers.py]: Why three asserts and not just one comparing the full value?

File: backend/tests/test_integration_address.py [hunk 2]
```diff
@@ -1401,23 +1439,24 @@ def test_missing_address(self, mock):
         self.assertRequestResponse(
             '/service/ou/{}/details/address'.format(unitid),
             [{
-                'address_type': address_class,
-                'name': 'Ukendt',
-                'value': addrid,
-                'error': 'no such address {!r}'.format(addrid),
+                'address_type': {
```
Comment (danchr) [inline backend/tests/test_integration_address.py]: Erm, the name of this test is `test_missing_address`, yet it now succeeds. Does it still test a properly missing address?

Comment (cmoesgaard) [reply backend/tests/test_integration_address.py]: It succeeds, yes, in the same way that the previous one did. We're unable to read the address from DAR but still return _something_ to be rendered in the UI.

Comment (danchr) [reply backend/tests/test_integration_address.py]: > It succeeds, yes, in the same way that the previous one did. We're unable to read the address from DAR but still return _something_ to be rendered in the UI.

Yeah, I saw that afterwards; I was a bit confused by it appearing slightly less fail-y.

Comment (cmoesgaard) [reply backend/tests/test_integration_address.py]: I omitted the `error` part of the response. As far as I could tell, we didn't use it for anything. We log the error as well, so we should be good :slightly_smiling_face:

**Pull request merged**